modules/snigelBidAdapter.js

```diff
@@ -104,9 +115,7 @@ export const spec = {
 registerBidder(spec);
 
 function getPage(bidderRequest) {
-  return (
-    getConfig(`${BIDDER_CODE}.page`) || deepAccess(bidderRequest, 'refererInfo.page') || window.location.href
-  );
+  return getConfig(`${BIDDER_CODE}.page`) || deepAccess(bidderRequest, 'refererInfo.page') || window.location.href;
```

Comment:
what is the goal of this override? do you not like the logic of refererInfo.page

Reply:
> what is the goal of this override? do you not like the logic of refererInfo.page

This is used for internal testing among other things to mock the page URL. In virtually all cases it just falls back to `refererInfo.page`